install/kubernetes/cilium/values.yaml

```diff
@@ -1711,6 +1711,10 @@ clustermesh:
         #                 time.
         # - certmanager:  This method use cert-manager to generate & rotate certificates.
         method: helm
+        # -- Node tolerations for pod assignment on nodes with taints
+        # Only used if method is "cronjob".
+        # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+        tolerations: []
```

Comment:
We're a bit inconsistent with the tolerations in Hubble. But given that `hubble.tls.auto` is used for more than just certgen, and this value is only relevant for `certgen`, I think we should add this value as `certgen.toleratons` instead.
It will be consistent with `certgen.podLabels` and `certgen.ttlSecondsAfterFinished`.

Reply:
I completely missed that. Changes committed 🚀